scripts/commit-msg.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+PATH=.venv/bin:"$PATH"
+
+git secrets --commit_msg_hook -- "$@"
```

Comment:
Yes, I know about the error. I don't know why because when executing `git secrets --aws-provider`, it's working.

Officially, it's git secrets, so we should use that, IMO.